pipeline/07-export.R

```diff
@@ -581,21 +575,46 @@ for (town in unique(assessment_pin_prepped$township_code)) {
         is.na(comp_pin_1_coord),
         NA,
         glue::glue(
-          '=HYPERLINK("[{workbook_name}]{comp_sheet_name}!',
-          '{comp_pin_1_coord}","{comp_pin_1}")'
+          '=HYPERLINK(@CELL("address",{comp_sheet_name}!{comp_pin_1_coord}),',
+          '"{comp_pin_1}")'
```

Comment:
Docs for the different `HYPERLINK` formats [here](https://support.microsoft.com/en-us/office/hyperlink-function-333c7ce6-c5ae-4164-9c47-7de9b76f577f#ID0EFD); the gist of this change is that by using the `CELL` function to dynamically return a reference to the comp sheet, the `HYPERLINK` address no longer depends on the name of the workbook. As such, I've moved the workbook name variable definition down in the sheet closer to where it is being used.